test/index.js

```diff
@@ -32,12 +32,22 @@ const changeCwd = function (cwd) {
   return process.chdir.bind(process, originalCwd)
 }
 
+// Move .next from sample project to current directory
+const moveNextDist = function () {
+  // Use copySync because cpx doesn't copy hidden files
```

Comment:
Do you mean dotfiles by "hidden files"?
If so, I believe `cpx` does support it (see [this issue](https://github.com/mysticatea/cpx/issues/45)). The problem might come from the globbing pattern instead. Most globbing libraries will ignore dotfiles with `*` and `**` unless some `dotfile` option is passed. However, `.*` will work (see [this comment](https://github.com/mysticatea/cpx/issues/45#issuecomment-418163164)).

Reply:
oh huh i was going off [this one](https://github.com/mysticatea/cpx/issues/34) but gotchu

Reply:
If that does not work, [`cpy`](https://github.com/sindresorhus/cpy) could also be used.

Reply:
ive tried several things to copy .next with cpx and none have worked for me, feel free to pull n try n prove me wrong !!!

Reply:
Did it work with `cpy` instead?

Reply:
nope

Reply:
The following commit fixes it with `cpy`: b099569
I haven't pushed it so you can review it first.

Reply:
i was def using cpy wrong, the word `dot` literally isnt even included in the readme

<img width="1182" alt="Screen Shot 2020-12-01 at 5 58 31 AM" src="https://user-images.githubusercontent.com/57350178/100732085-43c69500-339a-11eb-95e0-7a8b8fb20c78.png">

thanks for fixing!!!!!

Reply:
ahhhh i cherry-picked and your commit fails the test. im gonna rebase it out for now; could you update your branch and make sure the tests pass there and PR it into this branch?

Reply:
`dot` is part of `globby` which is mentioned [there](https://github.com/sindresorhus/cpy#options) :)

I fixed the bug and pushed to this PR :+1: